src/containers/Accounts/AccountTransactionsTable/styles.scss

```diff
@@ -277,3 +277,20 @@
     background: $gray-800;
   }
 }
+
+.warning {
```

Comment:
This style will have global scope.  Where are the other places warning is define?

Reply:
https://github.com/ripple/explorer/blob/0c3efd06a98f40d247995c0e38b8f485cb137942/src/containers/NoMatch/nomatch.scss#L35